### DISCUSSION TITLE
How do I get the onValueChange only when value was changed by the Slider itself?

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [chakra-ui/zag#1098] [Q&A] [answered]
I have a range slider that is semi-controlled by two Number Inputs.
So when changing the value of the number input, the onValueChange is triggered at the same time.
I would like to have something like onValueChangeBySlider to know when the user changed the value by the slider.

**Comment by segunadebayo** [comment, marked as answer]
This can be implemented in userland code. Not something we want to add to Zag.js

You might want to control both components' onValueChange and sync them to the same state values, them track the `onValueChange` for the slider.
